controllers/syncer/csi_node.go

```diff
@@ -181,7 +185,7 @@ func (s *csiNodeSyncer) ensureContainer(name, image string, args []string) corev
 		Args:         args,
 		Env:          s.getEnvFor(name),
 		VolumeMounts: s.getVolumeMountsFor(name),
-		Resources:    ensureDefaultResources(),
+		//Resources:    ensureDefaultResources(),
```

Comment:
Please remove the commented code and I guess after this PR, method `ensureDefaultResources()` has become unused, please confirm and do cleanup accordingly